### DISCUSSION TITLE
... so about that osu!mania update (mobile mania) Great on phones.. tablets, not so much.

### DISCUSSION THREAD

**Discussion by Ysabelle12** [ppy/osu#25557] [Ideas] [answered]
It's perfectly fine and passable for a phone, but not all of us use iPhones to play osu!lazer. It's not comfortable for some of us, and the performance isn't great once it throttles down.

Let's not forget about the iPad users, especially those with the 12.9-inch iPad Pro. Many of my friends use the larger iPad with a keyboard and a trackpad, or just straight up external keyboard. Seeing osu!mania fill up the entire screen isn't great because it's way too stretched and too distracting. Not to forget that it doesn't properly scale on 7 keys mania.
![IMG_0154](https://github.com/ppy/osu/assets/37205797/9b6f5bef-5a34-49aa-b6db-a75fd8611a99)

So, what I'm trying to say is, please give us an option to use the old style of osu!mania. Some of us view the iPad Pro as a direct laptop replacement, and not everyone prefers to use the touchscreen for mania; some of us use a keyboard.

Additionally, I've noticed that recently, keyboards no longer work for searching things on osu!lazer on iPad. I'm not sure why that is.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/25440

**Reply by Maryamejaz642** [reply]
The recent osu!mania update for mobile has been a game-changer, delivering an exceptional experience on phones. The gameplay feels fluid and responsive, enhancing the overall mobile gaming experience. However, on tablets, some users have reported challenges, noting that the optimization doesn't match the seamless performance on phones. It seems the developers might need to fine-tune the tablet interface to ensure a consistent and enjoyable gaming experience across all devices. Looking forward to further updates that address these tablet-specific issues! 🎮📱
